Automattic/wp-calypso
# Adding WPCourses education card to My Home

This PR adds a WP Courses card to the Learn and Grow section of My Home as discussed at pcbrnV-264-p2.

Here's a screenshot of the card in place:

<img width="695" alt="Screen Shot 2021-04-20 at 3 19 26 PM" src="https://user-images.githubusercontent.com/35781181/115452174-1618a600-a1ec-11eb-8ced-bc3669d70140.png">

The card is being added to these views:
* VIEW_DEFAULT
* VIEW_CLOUDFLARE
* VIEW_EARN_FEATURES
* VIEW_FIND_DOMAIN
* VIEW_CONNECT_ACCOUNTS
* VIEW_WEBINARS
* VIEW_PODCASTING

#### Testing instructions
* Patch your sandbox with D60187-code
* Update your hosts file to point public-api.wordpress.com to your sandbox
* Checkout this branch and start Calypso
* Force views for each view listed above but loading: http://calypso.localhost:3000/home/SITESLUG?dev=true&view=VIEW_CLOUDFLARE
* Click the link in the card and confirm that it behaves as expected.
